libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -160,6 +163,31 @@ void AP_DDS_Client::populate_static_transforms(tf2_msgs_msg_TFMessage& msg)
 
 }
 
+void AP_DDS_Client::update_topic(geometry_msgs_msg_PoseStamped& msg)
+{
+    update_topic(msg.header.stamp);
+    strcpy(msg.header.frame_id, ENU_FRAME_ID);
+
+    auto &ahrs = AP::ahrs();
+    WITH_SEMAPHORE(ahrs.get_semaphore());
+
+    Quaternion orientation;
+    if (ahrs.get_quaternion(orientation))
+    {
+        msg.pose.orientation.x = orientation[2];
```

Comment:
In the static transforms for GPS PR, I linked the documentation to the conventions as a comment and showed how it mapped. Can you do the same here? That will make it easier to validate your math through inspection. We will also need to validate the orientation, perhaps through Foxglove 3D by checking bank angles.

Reply:
Sure! Added the comments. Will work on validating the orientation